src/scala/com/github/johnynek/bazel_deps/AetherResolver.scala

```diff
@@ -61,13 +69,67 @@ class AetherResolver(servers: List[MavenServer], resolverCachePath: Path) extend
     s
   }
 
-  private val repositories =
-    servers.map { case MavenServer(id, t, u) =>
+  private val m2Home = new File(new File(System.getProperty("user.home")), "/.m2/")
+  private val m2SecurityXml = new File(m2Home, "settings-security.xml")
+
+  private val settingsDecrypter: SettingsDecrypter = {
```

Comment:
this seems like a separate PR, can we please separate from the warning/error on imports?